corehq/toggles/__init__.py

```diff
@@ -2240,6 +2240,13 @@ def _commtrackify(domain_name, toggle_is_enabled):
     help_link="https://confluence.dimagi.com/display/GS/FHIR+API+Documentation",
 )
 
+MGH_EPIC_STUDY = StaticToggle(
```

Comment:
(Note to self: Read all the code before you start to comment.)

Instead of using a feature flag, I'd advocate moving this code into `custom.mgh_epic`, and enable the [custom module in settings.py](https://github.com/dimagi/commcare-hq/blob/442e112bf39dfd0fa9b65e3feb9d1bd72a939fea/settings.py#L1962) for the project spaces you want.

Reply:
Thanks for taking a look, Norman. This has been in a holding pattern for a while, but I'm coming back to it to do some more work now. I can move this to a custom module.

Reply:
moved code here https://github.com/dimagi/commcare-hq/pull/35125/commits/ff0e133859fae86422db02bede84c949cb609f51